### PR TITLE
CDS-74891-whitelist AWS endpoints

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Added a note for whitelisting AWS endpoints

### CODE REVIEW

File: docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md [hunk 1]
```diff
@@ -45,7 +45,15 @@ The AWS [IAM Policy Simulator](https://docs.aws.amazon.com/IAM/latest/UserGuide/
...
+  * **Use IRSA:** Allows the Harness Kubernetes delegate in AWS EKS use a specific IAM role when making authenticated requests to resources. By default, the Harness Kubernetes delegate uses a ClusterRoleBinding to the **default** service account. Instead, with this option, you can use AWS [IAM roles for service accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to associate a specific IAM role with the service account used by the Harness Kubernetes delegate. 
+
+  :::info
+
+  Verify your firewall policy and make sure to whitelist all AWS endpoints for the services you're using. For more details, go to, [view AWS service endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html#view-service-endpoints).
```
Comment (roshnisarangadharan) [inline docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md:52]:
```suggestion
  Verify your firewall policy and make sure to whitelist all AWS endpoints for the services you're using. For more details, go to [view AWS service endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html#view-service-endpoints).
```

File: docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md [hunk 2]
```diff
@@ -45,7 +45,15 @@ The AWS [IAM Policy Simulator](https://docs.aws.amazon.com/IAM/latest/UserGuide/
...
-  * **Use IRSA:** Allows the Harness Kubernetes delegate in AWS EKS use a specific IAM role when making authenticated requests to resources. By default, the Harness Kubernetes delegate uses a ClusterRoleBinding to the **default** service account. Instead, with this option, you can use AWS [IAM roles for service accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to associate a specific IAM role with the service account used by the Harness Kubernetes delegate. This option requires modifications to the delegate YAML, as described below.
+  * **Use IRSA:** Allows the Harness Kubernetes delegate in AWS EKS use a specific IAM role when making authenticated requests to resources. By default, the Harness Kubernetes delegate uses a ClusterRoleBinding to the **default** service account. Instead, with this option, you can use AWS [IAM roles for service accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to associate a specific IAM role with the service account used by the Harness Kubernetes delegate.
```
Comment (StrawberryPad) [inline docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md:48]:
```suggestion
  * **Use IRSA:** Allows the Harness Kubernetes delegate in AWS EKS to use a specific IAM role when making authenticated requests to resources. By default, the Harness Kubernetes delegate uses a ClusterRoleBinding to the **default** service account. Instead, with this option, you can use AWS [IAM roles for service accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to associate a specific IAM role with the service account used by the Harness Kubernetes delegate. 
```

File: docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md [hunk 3]
```diff
@@ -45,7 +45,15 @@ The AWS [IAM Policy Simulator](https://docs.aws.amazon.com/IAM/latest/UserGuide/
...
-  * **Use IRSA:** Allows the Harness Kubernetes delegate in AWS EKS use a specific IAM role when making authenticated requests to resources. By default, the Harness Kubernetes delegate uses a ClusterRoleBinding to the **default** service account. Instead, with this option, you can use AWS [IAM roles for service accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to associate a specific IAM role with the service account used by the Harness Kubernetes delegate. This option requires modifications to the delegate YAML, as described below.
+  * **Use IRSA:** Allows the Harness Kubernetes delegate in AWS EKS use a specific IAM role when making authenticated requests to resources. By default, the Harness Kubernetes delegate uses a ClusterRoleBinding to the **default** service account. Instead, with this option, you can use AWS [IAM roles for service accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to associate a specific IAM role with the service account used by the Harness Kubernetes delegate. 
+
+  :::info
```
Comment (StrawberryPad) [inline docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md:50]: I thought we were moving to :::info note style? (May not be applicable to this module)

File: docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md (hunk 1 above)
Comment (StrawberryPad) [inline docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md]:
```suggestion
  Verify your firewall policy and make sure to whitelist all AWS endpoints for the services you're using. For more details, go to [view AWS service endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html#view-service-endpoints).
```

**Review by StrawberryPad — APPROVED** [review]

File: docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md (hunk 3 above)
Comment (roshnisarangadharan) [inline docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md:50]:
```suggestion
  :::info note
```

Comment (roshnisarangadharan) [reply docs/platform/7_Connectors/Cloud-providers/add-aws-connector.md]: added :::info note style.

**Review by anilchowdhury-harness — APPROVED** [review]

**Pull request merged**